cspell.json

```diff
@@ -141,6 +141,7 @@
     "AppDelegate.swift",
     "AppDelegate",
     "appid",
+    "apigatewayv",
```

Comment:
is the `v` portion of this a typo?

Reply:
not a typo, the cspell is complaining on the v1 rest api and v2 http package import for apigateway